mantle/kola/tests/rpmostree/deployments.go

```diff
@@ -190,9 +190,9 @@ func rpmOstreeUpgradeRollback(c cluster.TestCluster) {
 // This uses a dummy RPM that was originally created for the atomic-host-tests;
 // see: https://github.com/projectatomic/atomic-host-tests
 func rpmOstreeInstallUninstall(c cluster.TestCluster) {
-	var ahtRpmPath = "/var/home/core/aht-dummy.rpm"
-	var installPkgName = "aht-dummy-1.0-1.noarch"
-	var installBinName = "aht-dummy"
+	var epelRpmPath = "/var/home/core/epel-release-latest-9.noarch.rpm"
```

Comment:
we need to update the `sudo rpm-ostree install` below to use this variable.

Reply:
I have updated it